### DISCUSSION TITLE
Emotion make lots of tags when open editor times

### DISCUSSION THREAD

**Discussion by 4-1-1** [Milkdown/milkdown#736] [Q&A]
<img width="583" alt="image" src="https://user-images.githubusercontent.com/4401981/191440261-e73c7884-4a63-45bc-b764-324289ec7946.png">

And the browser going to slower and slower.
Is there an emotion option for it. I can't find it 😭

**Comment by Saul-Mirone** [comment]
This is a bug and should be fixed by https://github.com/Saul-Mirone/milkdown/commit/00b263024c8e435c39267c89dba988d32dcd5433.

For quick fix, you can call `emotion.flush()` when closing the editor.

**Reply by Saul-Mirone** [reply]
```typescript
editor.action((ctx) => {
  ctx.get(emotionCtx).flush();
});
```

**Reply by 4-1-1** [reply]
not work

**Reply by 4-1-1** [reply]
It's a development env bug.  Just wait you fix 😄
